addon/utils/locale.js

```diff
@@ -13,6 +15,10 @@ export default class Locale {
   //  3. Default `rtl` to `false`
   //  4. Ensure `pluralForm` is defined
   constructor(id, container) {
+    if (initializedLocales[id]) {
+      return initializedLocales[id];
+    }
```

Comment:
OK. I'll give #255 a shot first. I may try moving some of the `addTranslations` logic into `Locale` so the `I18nService` doesn't have to worry about state management.